src/Sylius/Component/Core/Test/Services/SecurityService.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+ 
+namespace Sylius\Component\Core\Test\Services;
```

Comment:
This class should definitely go to the bundle, it is coupled with 2 Symfony interfaces.